contracts/interfaces/IERC20W2612.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity =0.7.6;
+
+interface IERC20W2612 {
```

Comment:
Name not descriptive and adds non standard permit from tak, IERCWithPermit or IERCPermitAllowed some better examples